CMakeLists.txt

```diff
@@ -52,3 +74,10 @@ include (OpmLibMain)
 
 # Install build system files
 install(DIRECTORY cmake DESTINATION share/opm)
+
+else()
+###############################################
+# use other build system provided outside OPM
+###############################################
+include(ExternalBuild)
```

Comment:
Where is this file?

Reply:
In an external module that needs to be provided by CMAKE_MODULE_PATH.